src/main/java/io/jenkins/plugins/report/jtreg/main/comparator/FailedTests.java

```diff
@@ -12,7 +12,7 @@
 
 public class FailedTests {
     // function for getting failed tests into an ArrayList
-    private static ArrayList<String> getBuildFailedTests(File build) throws Exception {
+    private static ArrayList<String> getBuildFailedTests(File build, String exactTestsRegex) throws Exception {
```

Comment:
I think it is time to start to work on some html formater next to the lain text one. Ohterwise it will become very hard to add one in future

Reply:
I created a  very basic Formatters interface with plaintext, color text and html implementations, you can check it out.